src/fastapi_swagger2/utils.py

```diff
@@ -507,20 +507,22 @@ def get_swagger2(
         # output["definitions"] = {k: definitions[k] for k in sorted(definitions)}
         output["definitions"] = {}
         for k in sorted(definitions):
-            properties = definitions[k]["properties"]
-            for p in properties:
-                if "anyOf" in properties[p].keys():
-                    any_of = properties[p].pop("anyOf")
-                    if len(any_of) <= 2:
-                        for _any_of in any_of:
-                            if _any_of == {"type": "null"}:
-                                properties[p]["x-nullable"] = True
-                            else:
-                                properties[p].update(_any_of)
-                    else:
-                        logger.warning(
-                            f"fastapi_swagger2: Unable to handle anyOf in definitions {any_of}"
-                        )
+            if "properties" in definitions[k].keys():
+                properties = definitions[k]["properties"]
```

Comment:
@entropi What do you think of this to make it more Pythonic 

`properties = definitions[k].get("properties", [])`

Reply:
works for me, updated.